extras/repos/dave_sim.repos

```diff
@@ -1,65 +1,37 @@
 repositories:
-  dave/dave:
-    type: git
-    url: https://github.com/Field-Robotics-Lab/dave.git
-    version: master
-
-  dave/acoustic_msgs:
+  acoustic_msgs:
     type: git
     url: https://github.com/apl-ocean-engineering/acoustic_msgs.git
     version: main
-
-  dave/frl_msgs:
+  dave:
     type: git
-    url: https://github.com/Field-Robotics-Lab/frl_msgs.git
-    version: master
-
-  dave/gtri_based_sonar:
+    url: git@github.com:Field-Robotics-Lab/dave.git
+    version: bsb/update_2_fix_dockerfile
+  ds_msgs:
     type: git
-    url: https://github.com/Field-Robotics-Lab/gtri_based_sonar.git
+    url: https://bitbucket.org/whoidsl/ds_msgs.git
     version: master
-
-  dave/nps_uw_sensors_gazebo:
+  ds_sim:
     type: git
-    url: https://github.com/Field-Robotics-Lab/nps_uw_sensors_gazebo.git
+    url: https://bitbucket.org/whoidsl/ds_sim.git
     version: master
-
-  dave/nps_uw_multibeam_sonar:
+  eca_a9:
     type: git
-    url: https://github.com/Field-Robotics-Lab/nps_uw_multibeam_sonar.git
+    url: https://github.com/uuvsimulator/eca_a9.git
     version: master
-
-  dave/glider_hybrid_whoi:
+  frl_msgs:
     type: git
-    url: https://github.com/Field-Robotics-Lab/glider_hybrid_whoi.git
+    url: git@github.com:Field-Robotics-Lab/frl_msgs.git
     version: master
-
-  uuv_sim/eca_a9:
-    type: git
-    url: https://github.com/uuvsimulator/eca_a9.git
-    version: master
-
-  uuv_sim/rexrov2:
+  rexrov2:
     type: git
     url: https://github.com/uuvsimulator/rexrov2.git
     version: master
-
-  uuv_sim/uuv_manipulators:
+  uuv_manipulators:
```

Comment:
I believe this URL also needs to be `Field-Robotics-Lab`, and `version: noetic_updates`.

Reply:
None of my updates to the ds_sim and ds_msgs repos should affect Noetic one way or the other.  I still can't test with Noetic, though (still can't get the Docker image to work), so I can't guarantee anything.  @crvogt did most of the Noetic updates, so any commits that he made to those repos might be Noetic related.

Reply:
Very good point.  I think we've got out of sync w.r.t. which repos and which forks we are officially supporting.  I think @crvogt and @M1chaelM should be able to tell us what is the most recent for the change to Noetic.  I'll get that info and make any updates to this PR.